### ISSUE TITLE
Typing text with successive uppercase characters

### ISSUE THREAD

**Issue by ondrejhanslik** [moredip/Frank#222]
I have a `UITextView` with `UITextAutocapitalizationTypeSentences`
I am trying to type text `ABC`.

Normally it works but sometimes I get `AbC` or `ABc` instead. I guess there is a collision when the text view sets the keyboard to lowercase but KIF typer still sees it as uppercase.

**Comment by ondrejhanslik** [comment]
I have tried several workarounds - I tried to split the text by [A-Z] and then entered every sequence separately, inserting long delays. That doesn't work. Sometimes the uppercase letter is not typed at all. I guess the problem lies in the small delay between tapping "shift" and tapping the letter key. 0.05 delay is long enough for letters on the same keyplane but it's not enough for changing keyplanes.

**Comment by ondrejhanslik** [comment]
Now with iOS 7:

Keyboard typing attempt n. 1 (written text: "4qyaQp0a", expected: "CR16-100A")
Keyboard typing attempt n. 2 (written text: ".Rq6-100", expected: "CR16-100A")
Keyboard typing attempt n. 3 (written text: ".CR16-100A", expected: "CR16-100A")
Keyboard typing attempt n. 4 (written text: "crq6-100a", expected: "CR16-100A")

Not seeing the same problem on my local computer, only in jenkins builds - the animations probably take a bit longer there.

**Comment by dgmora** [comment]
@ondrejhanslik  Could you find a workaround for this? So far I wasn't able to find something reliable

**Comment by ondrejhanslik** [comment]
No, it must be fixed in frank core - increasing the delay after a keyplane switch.
